docs/Reference/Pantheon-API-Methods.md

```diff
@@ -3771,6 +3771,54 @@ None
     The `EEA` API methods are not enabled by default for JSON-RPC. Use the [`--rpc-http-api`](Pantheon-CLI-Syntax.md#rpc-http-api) 
     or [`--rpc-ws-api`](Pantheon-CLI-Syntax.md#rpc-ws-api) options to enable the `EEA` API methods.
 
+### eea_getPrivateTransaction
+
+Returns the private transaction if you are a participant; otherwise, null. Get the public transaction 
+using [`eth_getTransactionByHash`](#eth_gettransactionbyhash) with the transaction hash returned by 
+[`eea_sendRawTransaction`](#eea_sendrawtransaction) or [`eea_sendTransction`](https://docs.ethsigner.pegasys.tech/en/latest/Using-EthSigner/Using-EthSigner/#eea_sendtransaction). 
+
+**Parameters** 
+
+`data` - Value of `input` from the public transaction
+
+**Returns**  
+
+Object - [Transaction object](Pantheon-API-Objects.md#transaction-object), or `null` if not a participant in the private transaction. 
```

Comment:
It is not a `Transaction object` as specified in `Pantheon-API-Objects.md`

I think we should specify a `Private Transaction object` and link it here